frontend/images/material_lock.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="150px" height="150px" viewBox="0 0 150 150" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch -->
+    <title>Untitled</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Attention_000000"></g>
+        <g id="Lock_000000" fill-rule="nonzero" fill="#C1C1C1">
+            <path d="M75,0 C116.25,0 150,33.75 150,75 C150,116.25 116.25,150 75,150 C33.75,150 0,116.25 0,75 C0,33.75 33.75,0 75,0 Z M75,15 C42,15 15,42 15,75 C15,108 42,135 75,135 C108,135 135,108 135,75 C135,42 108,15 75,15 Z M75,36 C86.633335,36 96,45.3029479 96,56.8571429 L96,60.3333333 C99.85,60.3333333 103,63.4619048 103,67.2857143 L103,102.047619 C103,105.871429 99.85,109 96,109 L54,109 C50.15,109 47,105.871429 47,102.047619 L47,67.2857143 C47,63.4619048 50.15,60.3333333 54,60.3333333 L54,56.8571429 C54,45.3029479 63.3666668,36 75,36 Z M75,77.7142857 C71.15,77.7142857 68,80.8428571 68,84.6666667 C68,88.4904762 71.15,91.6190476 75,91.6190476 C78.85,91.6190476 82,88.4904762 82,84.6666667 C82,80.8428571 78.85,77.7142857 75,77.7142857 Z M75,42.952381 C67.0333332,42.952381 61,48.9446711 61,56.8571429 L61,60.3333333 L89,60.3333333 L89,56.8571429 C89,48.9446711 82.966665,42.952381 75,42.952381 Z" id="Combined-Shape"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file